panel/src/components/Dialogs/LicenseDialog.vue

```diff
@@ -6,45 +6,54 @@
 		@cancel="$emit('cancel')"
 		@submit="$emit('submit')"
 	>
-		<k-bar style="margin-bottom: var(--spacing-2)">
-			<h2 class="k-headline">
-				{{ $t("license") }}
-			</h2>
+		<k-stack>
+			<k-bar>
+				<h2 class="k-headline">
+					{{ $t("license") }}
+				</h2>
 
-			<k-button
-				:text="$t('remove')"
-				icon="trash"
-				size="xs"
-				variant="filled"
-				dialog="license/remove"
-			/>
-		</k-bar>
+				<k-button
+					:text="$t('remove')"
+					icon="trash"
+					size="xs"
+					variant="filled"
+					dialog="license/remove"
+				/>
+			</k-bar>
 
-		<div class="k-table">
-			<table style="table-layout: auto">
-				<tbody>
-					<tr>
-						<th data-mobile="true">{{ $t("type") }}</th>
-						<td data-mobile="true">{{ license.type }}</td>
-					</tr>
-					<tr v-if="license.code">
-						<th data-mobile="true">{{ $t("license.code") }}</th>
-						<td data-mobile="true" class="k-text">
-							<code>{{ license.code }}</code>
-						</td>
-					</tr>
-					<tr v-if="license.info">
-						<th data-mobile="true">{{ $t("status") }}</th>
-						<td data-mobile="true" :data-theme="license.theme">
-							<p class="k-license-dialog-status">
-								<k-icon :type="license.icon" />
-								{{ license.info }}
-							</p>
-						</td>
-					</tr>
-				</tbody>
-			</table>
-		</div>
+			<k-definitions>
+				<k-definition :term="$t('type')">
+					<p class="k-license-dialog-type">
+						{{ license.type }}
+						<k-button
+							icon="info"
+							link="https://getkirby.com/license"
```

Comment:
@lukasbestle I had the idea if we could add a feature to getkirby.com where we could resolve an activation date or similar to a license terms version, e.g. `https://getkirby.com/license?on=2025-10-15` redirecting to the license terms that were the current ones at that time point in time.

Reply:
I really like that. Could we already add that to the PR and then take care of it on the other side later?

Reply:
Thinking again about this, modifications to the license terms are binding also for previous purchases. So I think we should always link to the most recent terms.